### ISSUE TITLE
[RESEARCH] Internationalize small content pieces within elements.

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-web-components#1975]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Description of issue

Smaller tools like https://github.com/shoelace-style/localize#readme allow for customization of small pieces of a component library.

Larger tools like https://github.com/lit/lit/tree/main/packages/localize might imply that we should take a position on internationalization for our consumers as well.

Find a solution to covering the few cases where we're shipping content within our components that needs to be translated without surfacing superfluous properties for this action.

**Comment by jnurthen** [comment]
Any update on thoughts for this? It seems like one of the major obstacles to supporting accessibility well in languages other than English.

**Comment by jnurthen** [comment]
The following components would currently be broken when translated (probably not exhausative):
* Asset (default hardcoded but can be overridden)
* SplitButton (aria-label of split hardcoded to More)
* Dialog (hardcoded close button label)
* NumberField (hardcoded increase/decrease buttons)
* Picker (hardcoded visually hidden dismiss button)
* Search (hardcoded Reset button)
* Tag (hardcoded Remove button)
* Toast (hardcoded Information/Error/Success Icon & Close button)

**Comment by majornista** [comment]
Color Slider currently has the string "hue", which can be overwritten using the `label` prop: https://github.com/adobe/spectrum-web-components/blob/a447ba2714483619a17ab73e11efc79f9f815c84/packages/color-slider/src/ColorSlider.ts#L57

Color Area currently uses "saturation" and "luminosity", which can be overwritten using the `labelX` and `labelY` props respectively: https://github.com/adobe/spectrum-web-components/blob/a447ba2714483619a17ab73e11efc79f9f815c84/packages/color-area/src/ColorArea.ts#L53-L60

For Color Area, it may be better to include the color channel names as part of the `aria-valuetext` for each of the inputs, so that the user aware of which channel their interaction changed, and we should include additional strings:

- 'Color Picker' as the default `aria-label` for each of the `input[type="range"]` elements, which should be concatenated with whatever `label` the user provides, so that the control announces something like "Foreground Color Picker".
- '2d slider' as the `aria-roledescription` for each of the `input[type="range"]` elements, so that he user gets some indication of the keyboard behavior for the control and understands that keyboard interaction changes values in two dimensions.

This is similar to the way a Color Picker is labelled on Windows, and is what we do in the React-Spectrum color area. See https://react-spectrum.adobe.com/react-aria/useColorArea.html#internationalization

**Comment by castastrophe** [comment]
We have this work tracked now in Jira so I'm going to close. Please do reach out if you still need a status update on this feature.
